zeppelin-web-angular/e2e/tests/notebook/published/published-paragraph.spec.ts

```diff
@@ -138,4 +155,53 @@ test.describe('Published Paragraph', () => {
     await runButton.click();
     await expect(modal).toBeHidden();
   });
+
+  test('should show confirmation modal in React mode and allow running the paragraph', async ({ page }) => {
+    const { noteId, paragraphId } = testNotebook;
+
+    await test.step('Given I clear paragraph output in normal notebook view', async () => {
+      await publishedParagraphPage.navigateToNotebook(noteId);
+
+      const paragraphElement = page.locator('zeppelin-notebook-paragraph').first();
+      const paragraphResult = paragraphElement.locator('zeppelin-notebook-paragraph-result');
+
+      // Only clear output if result exists
+      if (await paragraphResult.isVisible()) {
+        const settingsButton = paragraphElement.locator('a[nz-dropdown]');
+        await settingsButton.click();
+
+        const clearOutputButton = page.locator('li.list-item:has-text("Clear output")');
+        await clearOutputButton.click();
+        await expect(paragraphResult).toBeHidden();
+      }
```

Comment:
I think test scenarios should be as deterministic as possible.
Since `beforeEach` creates a new notebook, we may not need a conditional `if` block here to clear the output.
It should be sufficient to assert that the paragraph result is hidden with something like `expect(paragraphResult).toBeHidden()`.